### DISCUSSION TITLE
How to skip  mounting some subdirs from project root ?

### DISCUSSION THREAD

**Discussion by srepn** [ddev/ddev#5167] [Q&A] [answered]
Hello!
Is it possible to use .dockerignore or another way to ignore specific subdirectory in project root ?
Thanks

**Comment by rfay** [comment]
Hi @sergiirepin Can you give a specific example of what you want and what "ignore" means? Do you want the code not to be available inside the container? If you can give a specific example maybe I can help. Also tell me what OS and Docker provider you're working with.

**Comment by srepn** [comment]
Hey @rfay, thanks for answer!
> Do you want the code not to be available inside the container?
yes

For example I have two vendor folders and I want to decide which one should be mounted to the container. Something like what.
But in general ability yo use .dockerignore seems good idea ?

**Reply by rfay** [reply]
.dockerignore is not about mounting or anything like that. It's a build-time file (when building images) to say what is to be left out of the image build.

The easiest thing for you to do given what you're suggesting is to mount both directories (include them as siblings in the project) but update a symlink to show which one you want to use. For example, you could have  `vendorone` and `vendortwo` directories, and `ln -s vendortwo vendor` to choose which one you want.

**Comment by srepn** [comment]
@rfay yeah, this is how it works for me right now :)
Both dirs are mounted by default. 
And I just remount vendor folder by my needs.

**Reply by rfay** [reply]
No need to remount, just symlink.

**Reply by rfay** [reply, marked as answer]
I should say that you could in fact separately mount the vendor directory using a `docker-compose.*.yaml` and it would work. But lots of work compared to just letting both of them live side-by-side and link the one you want.
